django/contrib/admin/static/admin/js/admin/RelatedObjectLookups.js

```diff
@@ -106,8 +106,15 @@
             let option = select.querySelector(`option[value="${objId}"]`);
 
             if (!option) {
+                const previousValue = $(select).val();
```

Comment:
Hello @shangxiao !

I belive Django is trying to remove jQuery as much as possible so I would not use its reference `$` in new code if possible.

Correct me if I'm wrong @felixxm 

Thanks!

Reply:
@mgaligniana I agree though the select uses select2 … I assumed this was a necessity. I should take a second look at interacting with the widget 👍

Reply:
> Correct me if I'm wrong @felixxm

Yes, we're trying to avoid `jQuery` as much as possible. On the other hand this patch is intended for backporting and `jQuery` is already extensively used in `RelatedObjectLookups.js` so I'm fine with this.